infra/modules/providers/azure/app-insights/README.md

```diff
@@ -1,26 +1,25 @@
 
-More information for Azure Application Insights can be found [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)
+More information for Azure Application Insights can be found [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview).
 
-A terraform module in Cobalt to provide Application Insights with the following characteristics:
+This directory contains a terraform module in Cobalt to create a new instance of Application Insights.
 
-- Ability to deploy Application Insights in the same resource group as the Service Plan.
-- Ability to deploy Application Insights in the same resource group location in which the Service Plan is deployed.
-- Also gives ability to specify the following for Application Insights based on the requirements:
-  - name : Specifies the name of the Application Insights component. Changing this forces a new resource to be created.
-  - application_type : Specifies the type of Application Insights to create. Valid values are ios for iOS, java for Java web, MobileCenter for App Center, Node.JS for Node.js, other for General, phone for Windows Phone, store for Windows Store and web for ASP.NET. Please note these values are case sensitive; unmatched values are treated as ASP.NET by Azure. Changing this forces a new resource to be created.
+The following characteristics can be specified by the user:
+
+  - name: Specifies the name of the Application Insights component. Changing this forces a new resource to be created.
+  - application_type: Specifies the type of Application Insights to create. Valid values are ios for iOS, java for Java web, MobileCenter for App Center, Node.JS for Node.js, other for General, phone for Windows Phone, store for Windows Store and web for ASP.NET. Please note these values are case sensitive; unmatched values are treated as ASP.NET by Azure. Changing this forces a new resource to be created.
+  - resource_group_name: Specifies the resource group where the Service Plan was deployed (so the Application Insights instance gets deployed to the same resource group)
   - tags : A mapping of tags to assign to the resource.
 
 Please click the [link](https://www.terraform.io/docs/providers/azurerm/r/application_insights.html) to get additional details on settings in Terraform for Azure Application Insights.
 
 ## Usage
 
-### Module Definitions
+Use of this App Insights module assumes that a resource group has already been created, and that an App Service Plan has already been deployed to it.  The Cobalt module to deploy an App Service Plan is located [here](infra/modules/providers/azure/service-plan).
 
-- Service Plan Module : infra/modules/providers/azure/service-plan
-- App Insights Module : infra/modules/providers/azure/app-insights
+Sample usage of the Service Plan and App Insights modules:
```

Comment:
It would be great to include usage guidance for cobalt templates. Not sure how often cobalt users would use a module outside of a template?

```
module "app_insight" {
  source                           = "../../modules/providers/azure/app-insights"
  service_plan_resource_group_name = "${azurerm_resource_group.svcplan.name}"
  appinsights_name                 = "${local.app_insights_name}"
}
```

Reply:
I think I understand what you're saying.  Will make some changes to the readme